yt_dlp/extractor/spankbang.py

```diff
@@ -157,6 +157,7 @@ def extract_format(format_id, format_url):
             'description': description,
             'thumbnail': thumbnail,
             'uploader': uploader,
+            'uploader_id': uploader_id,
```

Comment:
Add a test containing the new field. The current test looks dead, so you can probably replace it with a new one.